cmd/nerdctl/container/container_cp_unix.go

```diff
@@ -1,3 +1,5 @@
+//go:build unix
```

Comment:
Isn't this implied by the file name suffix?

Reply:
I think only GOOS/GOARCH values are recognized as "magic" suffix implying a build tag.

IIRC they tried to introduce the same semantic for `_unix` but it backfired and broke a bunch of projects so they only added the `// go:build unix` directive but not the file matching.